### DISCUSSION TITLE
I can't view screenshots attached to an old github thread

### DISCUSSION THREAD

**Discussion by bennetthaselton** [community/community#23787] [New to GitHub]
<p>In this thread:</p>
<p><a href="https://github.com/aws/aws-sdk-php/issues/441" target="_blank" rel="nofollow noopener">https://github.com/aws/aws-sdk-php/issues/441</a></p>
<p>the OP refers to “the screenshot I provided in my first post”.  (And apparently the screen shot must have uploaded successfully at the time, because one of the respondents replied: ‘I just noticed that a potential problem could be that “Id” under “Owner” should be uppercase: “ID”’, and that’s not referring to any of the text in the thread, so presumably it was in the screen shot.)</p>
<p>However, as I’m viewing that thread now, I cannot see “the screenshot” anywhere.  Am I missing something?</p>
<p>Or were these threads archived and/or moved over from a different location, and in the archival/migration process, the text was preserved but the image attachments got lost somehow?</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi bennetthaselton! You’re not missing anything, the images really have been removed. It seems that this change was made before we started recording and showing edit history on issues - sorry if that’s caused you any confusion!</p>
